imagick/magick_wand_prop.go

```diff
@@ -132,14 +132,14 @@ func (mw *MagickWand) GetImageArtifacts(pattern string) (artifacts []string) {
 // GetImageProfile Returns the named image profile.
 //
 //	name: Name of profile to return: ICC, IPTC, or generic profile.
-func (mw *MagickWand) GetImageProfile(name string) string {
```

Comment:
We shouldn't break the existing API. What if we rename this function to GetImageProfileBytes, and then make a GetImageProfile call GetImageProfileBytes and return a string?